presidio-anonymizer/tests/integration/test_engine.py

```diff
@@ -38,9 +43,30 @@ def test_when_anonymize_called_with_multiple_scenarios_then_expected_results_ret
     )
 
     try:
-        actual_anonymize_result = engine.anonymize(anonymizer_request_dict.get("text"),
-                                                   analyzer_results, anonymizers_config)
+        actual_anonymize_result = engine.anonymize(
+            anonymizer_request_dict.get("text"), analyzer_results, anonymizers_config
+        )
     except Exception as e:
         actual_anonymize_result = str(e)
 
     assert actual_anonymize_result == expected_anonymize_result
+
+
+def test_when_anonymize_with_encrypt_then_text_returned_with_encrypted_content():
+    unencrypted_text = "My name is "
+    expected_encrypted_text = "Chloë"
+    text = unencrypted_text + expected_encrypted_text
+    start_index = 11
+    end_index = 16
+    key = "WmZq4t7w!z%C&F)J"
+    analyzer_results = [RecognizerResult("PERSON", start_index, end_index, 0.8)]
+    anonymizers_config = {"PERSON": AnonymizerConfig("encrypt", {"key": key})}
+
+    actual_anonymize_result = AnonymizerEngine().anonymize(
+        text, analyzer_results, anonymizers_config
+    )
+
+    assert actual_anonymize_result[:start_index] == unencrypted_text
+    actual_encrypted_text = actual_anonymize_result[start_index:]
+    actual_decrypted_text = AESCipher.decrypt(key.encode(), actual_encrypted_text)
```

Comment:
Make sure encrypted text is not the decrypt one...

Reply:
Done